src/ducks/projects.js

```diff
@@ -32,7 +32,8 @@ const fetchProjects = () => {
       type: FETCH_PROJECTS,
     });
     const query = {
-      current_user_roles: ['owner', 'translator']
+      current_user_roles: ['owner', 'translator'],
+      include: ['workflows']
```

Comment:
This query fetches all projects, so including all workflows is probably a bad idea.